doc/README.md

```diff
@@ -4,30 +4,33 @@ This directory contains python-sphinx documentation source.
 
 ## How to compile
 
-```
+```sh
```

Comment:
Previously I met cases that "sh" doesn't work but "bash" works. Do you have any idea if "sh" is good enough?